src/HttpMessage/HttpServerResponse.cs

```diff
@@ -43,28 +45,6 @@ public static HttpServerResponse Create(Version httpVersion, HttpResponseStatus
             return new HttpServerResponse(httpVersion, status);
         }
 
-        public byte[] Content

```

Comment:
The ResetContentLength is no longer needed so it's changed to a normal property with getters / setters (see: https://github.com/tomkuijsten/restup/pull/56/files#diff-9f31c4b1d539ccd646a27f4248d2445aR23).